.circleci/config.yml

```diff
@@ -172,7 +172,7 @@ jobs:
         command: |
           apt-get -y update
           apt-get -y install postgresql-client-12
-          apt-get -y install unixodbc freetds-dev
+          apt-get -y install unixodbc-dev freetds-dev
```

Comment:
I assume this'll let it build. When it comes to testing runtime behavior, we'll need the MS driver, which is a separate "dynamically loaded" component given by the connection string to unixodbc.

Reply:
Gotcha. Can it be trivially installed via `apt`?

Reply:
There's a self-contained debian-slim Dockerfile that installs it here: https://github.com/fpco/odbc/blob/master/Dockerfile#L8-L16

I just did a `docker image build` with that Dockerfile and it works, so it's still good.